src/ucp/dt/dt.c

```diff
@@ -156,6 +156,11 @@ ucs_status_t ucp_dt_query(ucp_datatype_t datatype, ucp_datatype_attr_t *attr)
     switch (datatype & UCP_DATATYPE_CLASS_MASK) {
     case UCP_DATATYPE_CONTIG:
         attr->packed_size = ucp_contig_dt_elem_size(datatype);
+
+        if (attr->field_mask & UCP_DATATYPE_ATTR_FIELD_COUNT) {
```

Comment:
we can treat count as default 1 also for generic and iov

Reply:
Makes sense and then also fits to the API documentation.